lib/beacon/loader/page_module_loader.ex

```diff
@@ -56,6 +57,20 @@ defmodule Beacon.Loader.PageModuleLoader do
     """
   end
 
+  defp handle_event(%Page{path: path, events: events}) do
+    Enum.map(events, fn %PageEvent{} = event ->
+      if !Application.get_env(:beacon, :disable_safe_code, true) do
+        SafeCode.Validator.validate!(event.code, extra_function_validators: Beacon.Loader.SafeCodeImpl)
+      end
```

Comment:
We may want to move this to some helper. So we could do something like:

```elixir
Beacon.SafeCode.validate!(event.code)
```

This would already check for the event so we don't need to duplicate this and it would be easier to enable/disable. Wdyt?

Reply:
Not only that, but all retrievals of `:disable_safe_code` are in `if !` form. The config value should be renamed to something like `validate_code_safety`, the boolean value flipped and the bang dropped.

Reply:
Also +1 on the moving this to a function, if I weren't clear. We should probably do this in a separate PR.

Reply:
I'll create a follow up PR with this cleanup 👍

Reply:
@xn @felipe https://github.com/BeaconCMS/beacon/pull/27